Added unified image reconstruction description document.

jsy1001 opened the pull request:
Hi Eric,

As discussed, I have renamed my repository and added the unified description. Could you please merge?

Cheers,
John.

emmt commented:
Hi John,

Thats' done! Thank you.

If you agree with that, I will do the renaming trick: 'OI-Imaging' seems 
more appropriate now.

I will ask Jonathan Léger (in copy) to start some kind of web page for 
the JRA.

Cheers,

	Éric.

Le 23/06/2015 14:28, John Young a écrit :
> Hi Eric,
>
> As discussed, I have renamed my repository and added the unified
> description. Could you please merge?
>
> Cheers,
> John.
>
> ------------------------------------------------------------------------
>
>
>         You can view, comment on, or merge this pull request online at:
>
> https://github.com/emmt/OI-Interface/pull/4
>
>
>         Commit Summary
>
>   * Added unified image reconstruction description document.
>
>
>         File Changes
>
>   * *R* doc/interface/OI-Interface.bib
>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-0> (0)
>   * *R* doc/interface/OI-Interface.pdf
>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-1> (0)
>   * *R* doc/interface/OI-Interface.sty
>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-2> (0)
>   * *R* doc/interface/OI-Interface.tex
>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-3> (0)
>   * *A* doc/unified_desc/figs/fig1.pdf
>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-4> (0)
>   * *A* doc/unified_desc/figs/fig2a.pdf
>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-5> (0)
>   * *A* doc/unified_desc/figs/fig2b.pdf
>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-6> (0)
>   * *A* doc/unified_desc/figs/fig2c.pdf
>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-7> (0)
>   * *A* doc/unified_desc/figs/fig2d.pdf
>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-8> (0)
>   * *A* doc/unified_desc/figs/fig3a.pdf
>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-9> (0)
>   * *A* doc/unified_desc/figs/fig3b.pdf
>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-10> (0)
>   * *A* doc/unified_desc/figs/fig3c.pdf
>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-11> (0)
>   * *A* doc/unified_desc/figs/fig3d.pdf
>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-12> (0)
>   * *A* doc/unified_desc/figs/fig3e-alt.pdf
>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-13> (0)
>   * *A* doc/unified_desc/figs/fig4a.pdf
>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-14> (0)
>   * *A* doc/unified_desc/figs/fig4b.pdf
>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-15> (0)
>   * *A* doc/unified_desc/figs/fig4c.pdf
>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-16> (0)
>   * *A* doc/unified_desc/figs/fig4d.pdf
>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-17> (0)
>   * *A* doc/unified_desc/opt_col_sm.jpg
>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-18> (0)
>   * *A* doc/unified_desc/unified_desc.bib
>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-19> (577)
>   * *A* doc/unified_desc/unified_desc.pdf
>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-20> (0)
>   * *A* doc/unified_desc/unified_desc.tex
>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-21> (1702)
>
>
>         Patch Links:
>
>   * https://github.com/emmt/OI-Interface/pull/4.patch
>   * https://github.com/emmt/OI-Interface/pull/4.diff
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/emmt/OI-Interface/pull/4>.
>


-- 
                             Eric THIEBAUT
  CRAL / Observatoire de Lyon     | eric.thiebaut@univ-lyon1.fr
  9, avenue Charles Andre         | +33-(0)4 78 86 85 48
  F-69561 Saint Genis Laval Cedex | +33-(0)4 78 86 83 86 (fax)
      http://cral.univ-lyon1.fr/labo/perso/eric.thiebaut/

jsy1001 commented:
OK. I prefer OI -Imaging-JRA but its up to you.


On 23 June 2015 19:51:10 BST, "Éric Thiébaut" <notifications@github.com> wrote:
>Hi John,
>
>Thats' done! Thank you.
>
>If you agree with that, I will do the renaming trick: 'OI-Imaging'
>seems 
>more appropriate now.
>
>I will ask Jonathan Léger (in copy) to start some kind of web page for 
>the JRA.
>
>Cheers,
>
>	Éric.
>
>Le 23/06/2015 14:28, John Young a écrit :
>> Hi Eric,
>>
>> As discussed, I have renamed my repository and added the unified
>> description. Could you please merge?
>>
>> Cheers,
>> John.
>>
>>
>------------------------------------------------------------------------
>>
>>
>>         You can view, comment on, or merge this pull request online
>at:
>>
>> https://github.com/emmt/OI-Interface/pull/4
>>
>>
>>         Commit Summary
>>
>>   * Added unified image reconstruction description document.
>>
>>
>>         File Changes
>>
>>   * *R* doc/interface/OI-Interface.bib
>>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-0> (0)
>>   * *R* doc/interface/OI-Interface.pdf
>>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-1> (0)
>>   * *R* doc/interface/OI-Interface.sty
>>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-2> (0)
>>   * *R* doc/interface/OI-Interface.tex
>>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-3> (0)
>>   * *A* doc/unified_desc/figs/fig1.pdf
>>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-4> (0)
>>   * *A* doc/unified_desc/figs/fig2a.pdf
>>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-5> (0)
>>   * *A* doc/unified_desc/figs/fig2b.pdf
>>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-6> (0)
>>   * *A* doc/unified_desc/figs/fig2c.pdf
>>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-7> (0)
>>   * *A* doc/unified_desc/figs/fig2d.pdf
>>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-8> (0)
>>   * *A* doc/unified_desc/figs/fig3a.pdf
>>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-9> (0)
>>   * *A* doc/unified_desc/figs/fig3b.pdf
>>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-10> (0)
>>   * *A* doc/unified_desc/figs/fig3c.pdf
>>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-11> (0)
>>   * *A* doc/unified_desc/figs/fig3d.pdf
>>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-12> (0)
>>   * *A* doc/unified_desc/figs/fig3e-alt.pdf
>>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-13> (0)
>>   * *A* doc/unified_desc/figs/fig4a.pdf
>>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-14> (0)
>>   * *A* doc/unified_desc/figs/fig4b.pdf
>>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-15> (0)
>>   * *A* doc/unified_desc/figs/fig4c.pdf
>>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-16> (0)
>>   * *A* doc/unified_desc/figs/fig4d.pdf
>>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-17> (0)
>>   * *A* doc/unified_desc/opt_col_sm.jpg
>>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-18> (0)
>>   * *A* doc/unified_desc/unified_desc.bib
>>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-19> (577)
>>   * *A* doc/unified_desc/unified_desc.pdf
>>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-20> (0)
>>   * *A* doc/unified_desc/unified_desc.tex
>>     <https://github.com/emmt/OI-Interface/pull/4/files#diff-21>
>(1702)
>>
>>
>>         Patch Links:
>>
>>   * https://github.com/emmt/OI-Interface/pull/4.patch
>>   * https://github.com/emmt/OI-Interface/pull/4.diff
>>
>> —
>> Reply to this email directly or view it on GitHub
>> <https://github.com/emmt/OI-Interface/pull/4>.
>>
>
>
>-- 
>                             Eric THIEBAUT
>  CRAL / Observatoire de Lyon     | eric.thiebaut@univ-lyon1.fr
>  9, avenue Charles Andre         | +33-(0)4 78 86 85 48
>  F-69561 Saint Genis Laval Cedex | +33-(0)4 78 86 83 86 (fax)
>      http://cral.univ-lyon1.fr/labo/perso/eric.thiebaut/
>
>
>---
>Reply to this email directly or view it on GitHub:
>https://github.com/emmt/OI-Interface/pull/4#issuecomment-114606162

-- 
Sent from my Android phone with K-9 Mail. Please excuse my brevity.

emmt commented:
OK the OI-Interface GitHub project has been renamed as OI-Imaging-JRA

You probably have to edit .git/config to set the remote url:

------------------------------------------------------
[remote "origin"]
	url = git@github.com:emmt/OI-Imaging-JRA.git
------------------------------------------------------

	Éric.

Le 23/06/2015 21:44, John Young a écrit :
> OK. I prefer OI -Imaging-JRA but its up to you.
>
>
> On 23 June 2015 19:51:10 BST, "Éric Thiébaut" <notifications@github.com>
> wrote:
>  >Hi John,
>  >
>  >Thats' done! Thank you.
>  >
>  >If you agree with that, I will do the renaming trick: 'OI-Imaging'
>  >seems
>  >more appropriate now.
>  >
>  >I will ask Jonathan Léger (in copy) to start some kind of web page for
>  >the JRA.
>  >
>  >Cheers,
>  >
>  > Éric.
>  >
>  >Le 23/06/2015 14:28, John Young a écrit :
>  >> Hi Eric,
>  >>
>  >> As discussed, I have renamed my repository and added the unified
>  >> description. Could you please merge?
>  >>
>  >> Cheers,
>  >> John.
>  >>
>  >>
>  >------------------------------------------------------------------------
>  >>
>  >>
>  >> You can view, comment on, or merge this pull request online
>  >at:
>  >>
>  >> https://github.com/emmt/OI-Interface/pull/4
>  >>
>  >>
>  >> Commit Summary
>  >>
>  >> * Added unified image reconstruction description document.
>  >>
>  >>
>  >> File Changes
>  >>
>  >> * *R* doc/interface/OI-Interface.bib
>  >> <https://github.com/emmt/OI-Interface/pull/4/files#diff-0> (0)
>  >> * *R* doc/interface/OI-Interface.pdf
>  >> <https://github.com/emmt/OI-Interface/pull/4/files#diff-1> (0)
>  >> * *R* doc/interface/OI-Interface.sty
>  >> <https://github.com/emmt/OI-Interface/pull/4/files#diff-2> (0)
>  >> * *R* doc/interface/OI-Interface.tex
>  >> <https://github.com/emmt/OI-Interface/pull/4/files#diff-3> (0)
>  >> * *A* doc/unified_desc/figs/fig1.pdf
>  >> <https://github.com/emmt/OI-Interface/pull/4/files#diff-4> (0)
>  >> * *A* doc/unified_desc/figs/fig2a.pdf
>  >> <https://github.com/emmt/OI-Interface/pull/4/files#diff-5> (0)
>  >> * *A* doc/unified_desc/figs/fig2b.pdf
>  >> <https://github.com/emmt/OI-Interface/pull/4/files#diff-6> (0)
>  >> * *A* doc/unified_desc/figs/fig2c.pdf
>  >> <https://github.com/emmt/OI-Interface/pull/4/files#diff-7> (0)
>  >> * *A* doc/unified_desc/figs/fig2d.pdf
>  >> <https://github.com/emmt/OI-Interface/pull/4/files#diff-8> (0)
>  >> * *A* doc/unified_desc/figs/fig3a.pdf
>  >> <https://github.com/emmt/OI-Interface/pull/4/files#diff-9> (0)
>  >> * *A* doc/unified_desc/figs/fig3b.pdf
>  >> <https://github.com/emmt/OI-Interface/pull/4/files#diff-10> (0)
>  >> * *A* doc/unified_desc/figs/fig3c.pdf
>  >> <https://github.com/emmt/OI-Interface/pull/4/files#diff-11> (0)
>  >> * *A* doc/unified_desc/figs/fig3d.pdf
>  >> <https://github.com/emmt/OI-Interface/pull/4/files#diff-12> (0)
>  >> * *A* doc/unified_desc/figs/fig3e-alt.pdf
>  >> <https://github.com/emmt/OI-Interface/pull/4/files#diff-13> (0)
>  >> * *A* doc/unified_desc/figs/fig4a.pdf
>  >> <https://github.com/emmt/OI-Interface/pull/4/files#diff-14> (0)
>  >> * *A* doc/unified_desc/figs/fig4b.pdf
>  >> <https://github.com/emmt/OI-Interface/pull/4/files#diff-15> (0)
>  >> * *A* doc/unified_desc/figs/fig4c.pdf
>  >> <https://github.com/emmt/OI-Interface/pull/4/files#diff-16> (0)
>  >> * *A* doc/unified_desc/figs/fig4d.pdf
>  >> <https://github.com/emmt/OI-Interface/pull/4/files#diff-17> (0)
>  >> * *A* doc/unified_desc/opt_col_sm.jpg
>  >> <https://github.com/emmt/OI-Interface/pull/4/files#diff-18> (0)
>  >> * *A* doc/unified_desc/unified_desc.bib
>  >> <https://github.com/emmt/OI-Interface/pull/4/files#diff-19> (577)
>  >> * *A* doc/unified_desc/unified_desc.pdf
>  >> <https://github.com/emmt/OI-Interface/pull/4/files#diff-20> (0)
>  >> * *A* doc/unified_desc/unified_desc.tex
>  >> <https://github.com/emmt/OI-Interface/pull/4/files#diff-21>
>  >(1702)
>  >>
>  >>
>  >> Patch Links:
>  >>
>  >> * https://github.com/emmt/OI-Interface/pull/4.patch
>  >> * https://github.com/emmt/OI-Interface/pull/4.diff
>  >>
>  >> —
>  >> Reply to this email directly or view it on GitHub
>  >> <https://github.com/emmt/OI-Interface/pull/4>.
>  >>
>  >
>  >
>  >--
>  > Eric THIEBAUT
>  > CRAL / Observatoire de Lyon | eric.thiebaut@univ-lyon1.fr
>  > 9, avenue Charles Andre | +33-(0)4 78 86 85 48
>  > F-69561 Saint Genis Laval Cedex | +33-(0)4 78 86 83 86 (fax)
>  > http://cral.univ-lyon1.fr/labo/perso/eric.thiebaut/
>  >
>  >
>  >---
>  >Reply to this email directly or view it on GitHub:
>  >https://github.com/emmt/OI-Interface/pull/4#issuecomment-114606162
>
> --
> Sent from my Android phone with K-9 Mail. Please excuse my brevity.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/emmt/OI-Interface/pull/4#issuecomment-114636450>.
>


-- 
                             Eric THIEBAUT
  CRAL / Observatoire de Lyon     | eric.thiebaut@univ-lyon1.fr
  9, avenue Charles Andre         | +33-(0)4 78 86 85 48
  F-69561 Saint Genis Laval Cedex | +33-(0)4 78 86 83 86 (fax)
      http://cral.univ-lyon1.fr/labo/perso/eric.thiebaut/
